store/mongo/datastore_mongo.go

```diff
@@ -2760,6 +2763,41 @@ func (db *DataStoreMongo) FindNewerActiveDeployments(ctx context.Context,
 	return deployments, nil
 }
 
+// FindNewerActiveDeployment finds active deployments which were created
+// after createdAfter for which is either dynamic (requires post processing)
```

Comment:
We do not support dynamic deployments here (the code is OK, it's only the comment which is wrong).